backend/development.ini.sample

```diff
@@ -404,6 +404,17 @@ api.key = %(basic_setup.api_key)s
 # user.custom_properties.ui_schema_file_path = %(user.custom_properties.dir)s/ui.json
 # user.custom_properties.translations_dir_path = %(user.custom_properties.dir)s/locale
 
+###
+# AGENDA
+###
+
+# in Tracim's Frontend, when creating a new event, the description (note) of the event will be pre-filled with the content
+# of the file given in the following parameter. Leave empty if you don't want any pre-filling.
+; agenda.pre_filled_event.description_file_path =
+
+## example:
+# agenda.pre_filled_event.description_file_path = %(here)s/tracim_backend/templates/pre-filled_agenda_event_description.txt
+
```

Comment:
We can do this, yes.

Reply:
I agree with @PhilippeAccorsi : as there already configuration related to agendas prefixed by `caldav` the prefix of the new setting should have the same one.

Reply:
I think we should group the parameters together in the settings file, however I don't think the option should be placed under caldav because this feature is unrelated to caldav.

Do you confirm we should prefix it by `caldav` anyway @grignards?

Reply:
if you don't put the parameter in `caldav.radicale`, it should work as expected.

Reply:
Ok, let's go for it then